ui/bits/css/team/_show.scss

```diff
@@ -151,4 +151,10 @@ $section-margin-more: 5vh;
   &__log {
     margin-bottom: 2em;
   }
+
+  @media (max-width: 1200px) {
```

Comment:
please use existing media queries instead of hard-coding new ones

- ui/common/css/abstract/_media-queries.scss
- search `include mq-` to view lots of examples